services/ui-src/src/measures/2022/FUMCH/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect } from "react";
+import { useFormContext } from "react-hook-form";
+import * as CMQ from "measures/2021/CommonQuestions";
```

Comment:
```suggestion
import * as CMQ from "measures/2022/CommonQuestions";
```
HEDIS options are incorrect. Should be as simple as updating this import.

current:
![Screen Shot 2022-06-14 at 11 49 20 AM](https://user-images.githubusercontent.com/22647707/173620946-a8d24773-02eb-4766-957e-9b533e996846.png)

expected:
![Screen Shot 2022-06-14 at 11 50 51 AM](https://user-images.githubusercontent.com/22647707/173621166-268569c3-8200-4cc7-ae9a-026c26ef7951.png)